pages/docs/comparison.mdx

```diff
@@ -91,7 +91,26 @@ author your own light and dark mode experience across your application.
 
 ## How is Chakra different from Material UI?
 
-// TO DO (Feel free to submit a PR)
+The Material UI React library provides users with a handful of UI layout tools 
+but is most famous for its large breadth of pre-styled UI components to which 
+developers can apply custom styles to override out-of-the-box base styles. Chakra 
+UI is a more robust, layout-focused library that also provides developers with UI 
+components similar to those that Material UI provides, but has a greater focus on 
+the creation of flexible, composable, and scalable code. 
+
+A core concept to be considered when comparing the two frameworks is 
+'Ease of Modification'. Comparatively, Material UI adds far more classes to 
+individual HTML tags related to exported components and layouts, forcing developers 
```

Comment:
Chakra UI might benefit from clarifying this part. As far as I know, there are no correlations between the number of class names on a DOM node and the override of base styles. I would imagine other readers will come to the same conclusion. What's the idea behind it? Maybe it's about CSS specificity?